### PR TITLE
docs(user-mgt): restructure user management documentation

### PR DESCRIPTION
**Pull request by imShakil**
### Prepare

- [x] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [x] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

#### Target issue
  #9470 
  
 
  

-------------------
### Test and Document the changes
- [x] Static code analysis has been run locally and issues have been fixed
- [x] Relevant unit and integration tests have been added/updated
- [x] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc)

Please check the below before submitting your PR. The PR will not be merged if there are no commits that start with `docs:` to indicate documentation changes or if the below checklist is not selected.
- [x] **I confirm that there is no impact on the docs due to the code changes in this PR.**

### CODE REVIEW

**Review by mo-auto — APPROVED** [review]

**Comment by ossdhaval** [comment]
@imShakil Please check the impact on below and fix if needed.
- navigation (mkdocs.yml)
- links from other documents to the changed documents in this PR

**Comment by imShakil** [comment]
> @imShakil Please check the impact on below and fix if needed.
> 
> * navigation (mkdocs.yml)
> * links from other documents to the changed documents in this PR

These are okay.

File: docs/janssen-server/scim/config.md [hunk 1]
```diff
@@ -46,13 +46,17 @@ These contents should be then passed to the `patch-scim-config` operation, e.g.
 
 To retrieve the current server configuration using TUI proceed as below:
 
-1. Launch TUI, e.g. by running `jans tui`, and follow the prompts
-1. Highlight the SCIM tab using your keyboard's left/right arrow key
-1. Highlight the "Get Scim Configuration" button using the tab key
-1. Press enter
+1. Launch TUI, e.g. by running `python3 /opt/jans/jans-cli/jans_cli_tui.py`, and follow the prompts
```
Comment (ossdhaval) [inline docs/janssen-server/scim/config.md]: To run TUI, we are now using `jans tui` wrapper command. So the document should use the same.

File: docs/janssen-server/scim/custom-attributes.md [hunk 2]
```diff
@@ -5,46 +5,38 @@ tags:
...
-![output-json](https://github.com/JanssenProject/jans/assets/43112579/41804347-4084-4bb4-8bc5-05fc220ae394)
+![output-json](../../assets/scim-attribute-retrieve.png)
 
 In the JSON response, your new added attribute will appear.
 
 To customize the URI associated to the extension (whose default value is `urn:ietf:params:scim:schemas:extension:gluu:2.0:User`
 ), you can use TUI:
 
-* Navigate to `SCIM` using Jans TUI `jans tui`
+* Navigate to `SCIM` using Jans TUI `/opt/jans/jans-cli/jans_cli_tui.py`
```
Comment (ossdhaval) [inline docs/janssen-server/scim/custom-attributes.md]: same as the previous comment, we should now be using new `jans` wrapper command. That is `jans tui`.

**Review by ossdhaval — CHANGES_REQUESTED** [review]

**Comment by imShakil** [comment]
Thanks for the finding @ossdhaval  I have add commit regarding on those changes.

**Review by ossdhaval — APPROVED** [review]

**Review by mzico — APPROVED** [review]

**Pull request closed without merging**